doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -429,6 +429,19 @@ To build with the |nRFVSC|, specify ``-DSNIPPET="nrf91-modem-trace-uart" [...]``
 
 See :ref:`cmake_options` for more details.
 
+.. _tfm_enable_share_uart:
+
+Shared UART for application and TF-M logging
+============================================
+
+If you want to activate TF-M logging while having modem traces enabled, it can be useful to direct the TF-M logs to the UART (UART0) used by the application.
```

Comment:
```suggestion
If you want to activate TF-M logging while having modem traces enabled, it can be useful to direct the TF-M logs to the UART (**UART0**) used by the application.
```